docs/development/release.md

```diff
@@ -10,6 +10,15 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
 
 ### 1. Create a branch for the new release
 
+#### From Master
+
+If creating a new minor version release from master, create a branch titled `release-MAJOR.MINOR.x` (e.g. release-1.14.x) as a feature branch from master to allow us to easily backport patches to that version.  Then create a release branch from it.
+
+The name is not important, but `release-x.y.z` seems like a good choice.
```

Comment:
I think we can remove this line?

Reply:
I guess maybe something like if creating a new patch version, create a release branch off of the major version release branch.

Reply:
I went ahead and removed the name suggestion line, it does seem extraneous at this point.